### ISSUE TITLE
Cat synonyms?

### ISSUE THREAD

**Issue by franknic** [monarch-initiative/vertebrate-breed-ontology#65]
While attempting to map OMIA cat breeds to VBO, a question arose to do with synonyms:
Are the three "Non Recognized" entries (VBO:0100175, VBO:0100176, VBO:0100177) equivalent to "Domestic"? If so, can "Domestic" be entered as a synonym for each entry?

**Comment by ImkeTammen** [comment]
I wonder if Bianca Waud could advise on this?

**Comment by sabrinatoro** [comment]
@katiermullen , you are probably the best person who know about the cat data. Can you take a look at this issue, please? Thank you.

**Comment by katiermullen** [comment]
Yes, how to handle "unrecognized breeds" is an outstanding issue. This includes non recognized, domestic shorthair, domestic longhair, domestic medium hair, domestic hairless, companion cat, etc. I will propose a strategy to address this issue. Thank you for bringing this up.

**Comment by franknic** [comment]
Thank you, all. Yes, we should ask Bianca (and Leslie). In her original lists, Leslie included the three "Non Recognised" categories now in VBO, and two of the four "Household Pet" categories now in VBO. She also included three "Domestic" categories not yet in VBO:
Domestic Longhair (DLH)
Domestic Mediumhair (DMH)
Domestic Shorthair (DSH)
VBO also includes "House Cat Longhair" and "House Cat Shorthair".
From an OMIA point of view, in one way or another it would be great to have "Domestic" appearing somewhere in VBO, even if just as a synonym: "Domestic" is a very common term used in refereed papers to categorise cats (it appears in 28 feline OMIA phenes).
I quite like the "Non Recognised" categories because those words are an accurate reflection of the reality. My understanding is that "Domestic" would be a good synonym for "Non Recognised". But I may be wrong! Leslie and Bianca are best placed to provide advice on this. 
Finally, to follow up a point made at our most recent meeting, it would be fine for Leslie to see the current version of VBO, especially given that she created her breed lists without providing any genetic evidence.

**Comment by katiermullen** [comment]
Plan is to: 
1. Merge some of the non recognized cat "breeds" that are already in VBO and should really be synonyms. 
2. Include domestic longhair, domestic shorthair, and domestic medium hair as synonyms of the merged "non-recognized" cats where appropriate. 
3. Having the "domestic" term appear is deemed important as the vast majority of the cat population fall into this group and this is common terminology in the literature and for veterinarians/pet owners.

**Comment by franknic** [comment]
Thank you very much, Katie
That sounds like a good strategy to me. Have you checked with Leslie? It is really important that her feedback has been sought in these early stages. In that same context, Imke and I would be very pleased to check with our cat colleague, Bianca (mentioned previously by Imke). The greater the early feedback we get from people like Leslie and Bianca, the greater will be the utility and acceptance of VBO.

**Comment by franknic** [comment]
I should have added that Imke and I would be very pleased to do anything to help us receive feedback from Leslie and Bianca.

**Comment by katiermullen** [comment]
@ImkeTammen I am working on aligning some of the cat breed names. Do you currently use any of these terms in OMIA?

VBO:0100118 | House Cat Longhair (Cat)
VBO:0100119 | House Cat Shorthair (Cat)
VBO:0100120 | Household Pet (Cat)
VBO:0100121 | Household Pet Kitten (Cat)
VBO:0100122 | Household Pet Longhair (Cat)
VBO:0100123 | Household Pet Shorthair (Cat)
VBO:0100175 | Non Recognized Hairless (Cat)
VBO:0100176 | Non Recognized Longhair (Cat)
VBO:0100177 | Non Recognized Shorthair (Cat)

**Comment by katiermullen** [comment]
Thanks! I'll work on organizing the terms and send them to you for review. Do you have a source for Japanese domestic?

**Comment by franknic** [comment]
I think Imke is right.  I've found an old paper that may be worth citing:	https://doi.org/10.11238/jmammsocjapan1952.1.75
It reinforces the idea that this is not a breed, but simply refers to domesticated cats in Japan.

**Comment by katiermullen** [comment]
Thanks! We have another request to add domesticated cats from Japan. The reference is very helpful!

**Comment by katiermullen** [comment]
See https://docs.google.com/presentation/d/1ddqrOVbjtfcgGwH31tGgrPgMkgYPMwqdcQxZyoXoeb8/edit#slide=id.g27fcba10135_0_10
